src/common/telemetry/src/metrics.rs

```diff
@@ -12,11 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! query engine metrics
-
 pub static METRIC_PARSE_SQL_ELAPSED: &str = "query.parse_sql_elapsed";
 pub static METRIC_PARSE_PROMQL_ELAPSED: &str = "query.parse_promql_elapsed";
 pub static METRIC_OPTIMIZE_LOGICAL_ELAPSED: &str = "query.optimize_logicalplan_elapsed";
 pub static METRIC_OPTIMIZE_PHYSICAL_ELAPSED: &str = "query.optimize_physicalplan_elapsed";
 pub static METRIC_CREATE_PHYSICAL_ELAPSED: &str = "query.create_physicalplan_elapsed";
 pub static METRIC_EXEC_PLAN_ELAPSED: &str = "query.execute_plan_elapsed";
+
+pub const THREAD_NAME_LABEL: &str = "thread.name";
```

Comment:
We should define metrics in each crate. The common-telemetry crate only provides utilities to add metrics

Reply:
+1